zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterOutput.java

```diff
@@ -229,8 +229,11 @@ public void write(int b) throws IOException {
               return;
             }
           }
-          // not a defined display system
+          // not a defined display system, probably a spell type
+          setType(InterpreterResult.Type.SPELL);
```

Comment:
We can't assume that It's always the spell type.  Since log can include random text. 
What do you think of?